apps/theming-designer/src/components/SemanticSlotsDetailsList.tsx

```diff
@@ -0,0 +1,129 @@
+import * as React from 'react';
+
+import {
+  DetailsList,
+  DetailsRow,
+  IDetailsRowStyles,
+  IDetailsRowProps,
+  IColumn,
+  IGroup,
+  SelectionMode
+} from 'office-ui-fabric-react/lib/DetailsList';
+import { ITheme } from 'office-ui-fabric-react/lib/Styling';
+import { SemanticColorSlots, IThemeRules } from 'office-ui-fabric-react/lib/ThemeGenerator';
+
+export interface ISemanticSlotsDetailsListProps {
+  slotNames: string[];
+  noneSlots: JSX.Element[];
+  neutralSlots: JSX.Element[];
+  softSlots: JSX.Element[];
+  strongSlots: JSX.Element[];
+}
+
+interface ISemanticSlotsDetailsList {
+  key: string;
+  slotName: string;
+  noneSlot: JSX.Element;
+  neutralSlot: JSX.Element;
+  softSlot: JSX.Element;
+  strongSlot: JSX.Element;
+}
+
+export const SemanticSlotsDetailsList: React.StatelessComponent<ISemanticSlotsDetailsListProps> = (
+  props: ISemanticSlotsDetailsListProps
+) => {
+  let bodyIndex = 13;
+  let linkIndex = 17;
+  let buttonIndex = 42;
+  let inputIndex = 56;
+  let listIndex = 63;
+  let menuIndex = 71;
+
+  let items: ISemanticSlotsDetailsList[] = [];
+  let groups: IGroup[] = [];
+  let columns: IColumn[] = [];
+
+  // const onRenderRow = (detailsRowProps: IDetailsRowProps | undefined): JSX.Element => {
+  // };
```

Comment:
Unused comment.

Reply:
Added row styling so it's transparent on hover.